samples/extensions/raytracing_extended/raytracing_extended.cpp

```diff
@@ -872,8 +877,8 @@ void RaytracingExtended::create_descriptor_sets()
 {
 	std::vector<VkDescriptorPoolSize> pool_sizes = {
```

Comment:
Looking at `VkDescriptorSetLayoutCreateInfo` I count:
- 1x `VK_DESCRIPTOR_TYPE_ACCELERATION_STRUCTURE_KHR`
- 1x `VK_DESCRIPTOR_TYPE_STORAGE_IMAGE`
- 5x `VK_DESCRIPTOR_TYPE_STORAGE_BUFFER`
- 1x `VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER`
- `raytracing_scene->imageInfos.size()`x `VK_DESCRIPTOR_TYPE_COMBINED_IMAGE_SAMPLER`

Reply:
Oh, thanks right; should engage brain before commit.

Reply:
I know the feeling :) Looks good now!